Lab 1/Lab1/IOSystem/Readers/Reader.cs

```diff
@@ -0,0 +1,98 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Lab1.LogsAndExceptions;
+
+namespace Lab1
+{
+    class Reader
```

Comment:
Access modificators should be set in all project classes, methods, and interfaces.